Client/Frontend/Widgets/LoginTableViewCell.swift

```diff
@@ -55,6 +72,19 @@ class LoginTableViewCell: UITableViewCell {
         return imageView
     }()
 
+    /// Override the default accessibility label since it won't include the description by default
+    /// since it's a UITextField acting as a label.
+    override var accessibilityLabel: String? {
+        get {
+            return "\(highlightedLabel.text ?? ""), \(descriptionLabel.text ?? "")"
```

Comment:
Is it common for password fields to speak the obscured password in VoiceOver? It seems like that could be a security problem if passwords were read aloud just because the field is focused, though I'm not familiar with the iOS convention.